src/sentry/grouping/parameterization.py

```diff
@@ -0,0 +1,368 @@
+import dataclasses
+import re
+from collections import defaultdict
+from collections.abc import Callable, Sequence
+from functools import lru_cache
+
+import tiktoken
+
+__all__ = [
+    "ParameterizationCallable",
+    "ParameterizationCallableExperiment",
+    "ParameterizationExperiment",
+    "ParameterizationRegex",
+    "ParameterizationRegexExperiment",
+    "Parameterizer",
+    "UniqueIdExperiment",
+]
+
+
+@dataclasses.dataclass
+class ParameterizationRegex:
+
+    name: str  # name of the pattern also used as group name in combined regex
+    raw_pattern: str  # regex pattern w/o matching group name
+    lookbehind: str | None = None  # positive lookbehind prefix if needed
```

Comment:
```suggestion
    look_behind: str | None = None  # positive look_behind prefix if needed
```
Your call if it makes sense or not.

Reply:
In context of regex "lookbehind" and "lookahead" are the standard terms. 

https://www.regular-expressions.info/lookaround.html